README.rst

```diff
@@ -0,0 +1,45 @@
+==============
+ twikiget
+==============
+
+.. .TODO uncomment when active
+
+.. .image:: https://img.shields.io/pypi/pyversions/twikiget.svg
+  .:target: https://pypi.org/pypi/twikiget
+
+.. .image:: https://img.shields.io/travis/cernanalysispreservation/twikiget.svg
+  .:target: https://travis-ci.org/cernanalysispreservation/twikiget
+
+.. .image:: https://readthedocs.org/projects/docs/badge/?version=latest
+   .:target: https://twikiget.readthedocs.io/en/latest/?badge=latest
+
+.. .image:: https://img.shields.io/coveralls/cernanalysispreservation/twikiget.svg
+   .:target: https://coveralls.io/r/cernanalysispreservation/twikiget
+
+.. image:: https://img.shields.io/github/license/cernanalysispreservation/twikiget.svg
+   :target: https://github.com/cernanalysispreservation/twikiget/blob/master/LICENSE
+
+About
+-----
+
+twikiget is a tool to download twiki pages and archive them in `.warc` format.
+It is a wrapper for `wget <https://www.gnu.org/software/wget/>`_ so it includes
+all the features of it and makes it easier to work with twiki pages.
+
+
+Features
+--------
+
+- download requested page with all dependecies and attachements
```

Comment:
- download and archive specific TWiki page and all its attachments 
- create WARC files for long-term preservation purposes
- save local cache for faster and periodic reprocessing 
- (planned) extract specific metadata from TWiki document markup according to configurable templates